habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/BehaveBddTestMojo.java

```diff
@@ -83,6 +98,17 @@ public void doExecute() throws MojoExecutionException, MojoFailureException {
             executeBehaveTestArgs
                     .addAll(Arrays.asList("run", BEHAVE_PACKAGE, getCanonicalPathForFile(behaveDirectory)));
 
+            if (outputCucumberStyleTestReports) {
+                poetryHelper.installDevelopmentDependency(BEHAVE_CUCUMBER_FORMATTER);
+                executeBehaveTestArgs.add("--format=behave_cucumber_formatter:PrettyCucumberJSONFormatter");
+                executeBehaveTestArgs.add("--outfile=dist/cucumber-reports/cucumber.json");
```

Comment:
this installs a dependency that formats and creates the cucumber.json needed to create the final html report